app/modules/events/card.tsx

```diff
@@ -0,0 +1,17 @@
+import type { Event } from '@prisma/client';
+
+export function EventCard({ name, date, description, imgUrl, imgAlt, location }: Event) {
+  return (
+    <div className="grid grid-cols-3 gap-8 p-4 bg-primary text-white rounded-xl">
+      <div className="">
```

Comment:
Div may not be needed if no classes?

Reply:
in the future, there should be a background shape of a planet, which this div would be needed for the grid, but I do agree, there is no need for one currently.